db/migrate/20210910124918_create_join_table_questions_tags.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableQuestionsTags < ActiveRecord::Migration[6.1]
+  def change
+    create_join_table :questions, :tags do |t|
+      # t.index [:question_id, :tag_id]
```

Comment:
why commented

Reply:
these were auto generated as commented
and I am not sure if we would them, so I left them as it is